### DISCUSSION TITLE
Unable to retrieve password

### DISCUSSION THREAD

**Discussion by javaCR7** [requarks/wiki#7269] [Error / Bug Report]
When retrieving the password, prompt "The mail configuration is incomplete or invalid
How can I retrieve my password?
![image](https://github.com/user-attachments/assets/f5073878-1c39-4378-a54f-e0419a948bed)

**Comment by wikijsuser** [comment]
Hi Team,

I have the same issue as well. Can't remember either the username or even the password,

Thanks,

wikijsuser
